Project_One.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+
+<html>
+     <head>
+
+        <meta charset = "UTF-8">
+        <link rel = "stylesheet"
+        type = "text/css"
+        href = "Trash.css" />
+
+     </head>
+   
+        <body>
+
+            <p>Click Me Hard</p>
+
+            <button id="grading-element-one" onclick="TouchMe()">Touch Me</button>
+
+            <script>
+            function TouchMe(){
+                var touch = document.createElement("i");
+                touch.innerText = " ClickMeHarder ";
+                touch.setAttribute("id","grading-element-one-consequence")
+                document.body.appendChild(touch)
+            }
+            
+            
+            </script>
+
+            <p><a id="grading-element-two" href="TouchingMe.html">TouchingMe</a> This is my touching me location </p>
+
+                
+            <button id="grading-element-three" onclick="DontTouchMe()">DontTouchMe</button>
+
+            <script>        
+
+            function DontTouchMe(){
+                var click = document.createElement("i");
+                click.setAttribute("id","grading-element-three-consequence")

```

Comment:
almost...it needs to _start_ in the DOM, with `visibility: hidden`. Then in the function `DontTouchMe()`, set `.style.visibility = "visibile"`